docs/tutorials/lerobot.rst

```diff
@@ -0,0 +1,23 @@
+=======
+LeRobot
+=======
+
+This page contains tutorials for using the Trossen Arm with LeRobot for Imitation Learning.
+
+.. note::
+
+    You should have a **matching version** of **Trossen Arm Firmware** on your controller and the **Trossen Arm Python** package installed on your computer.
+    For more information about firmware update follow the `Trossen Arm Firmware Update <add link>`_.
```

Comment:
Sorry, let's actually remove this note. We'll have a note that makes this point clear further up in the docs.